### DISCUSSION TITLE
V3 Livewire support?

### DISCUSSION THREAD

**Discussion by cieplik206** [alpinejs/alpine#1426] [4. General]
Thanks Caleb for new Alpine....

but Am I doing something wrong or Livewire is not yet supported ?

`Alpine Expression Error: $wire is not defined`

Is there a way I can access Livewire data from alpine ?

**Comment by SimoTod** [comment]
Alpine v3 has a new way to register magic helpers like wires, I think you need to wait for Caleb to release the relevant update for livewire or plugin for Alpine.

**Comment by calebporzio** [comment]
Sorry, I forgot to announce that - Livewire doesn't currently support V3 and I hope to get support included in the next few weeks.

**Comment by joshhanley** [comment]
Support for Alpine V3 has been added in Livewire 2.5.0 today 🙂

**Reply by kurucu** [reply]
The fastest few weeks ever!
